packages/snaps-rpc-methods/src/restricted/getBip32Entropy.ts

```diff
@@ -31,6 +32,14 @@ export type GetBip32EntropyMethodHooks = {
    * @returns A promise that resolves once the extension is unlocked.
    */
   getUnlockPromise: (shouldShowUnlockRequest: boolean) => Promise<void>;
+
+  /**
+   * Get the cryptographic functions to use for the client. This may return an
+   * empty object to fall back to the default cryptographic functions.
+   *
+   * @returns The cryptographic functions to use for the client.
+   */
+  getClientCryptography: () => CryptographicFunctions;
```

Comment:
For the extension, do we wanna just do `getClientCryptography: () => ({])` or potentially `getClientCryptography: () => null` 🤔

Reply:
I was thinking an empty object, but `key-tree` takes `undefined` too, so I could add that to the return type.

Reply:
Might be slightly more clean than forcing return of an empty object